robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp

```diff
@@ -114,6 +114,27 @@ namespace gazebo
 
     createGenericTopics(node_name);
 
+    if (_sdf->HasElement("p")){
+      kp = _sdf->GetElement("p")->Get<double>();
+      RCLCPP_INFO(ros_node_->get_logger(), "Taking kp value of [%s] from plugin", std::to_string(kp).c_str());
+    }else{
+      RCLCPP_INFO(ros_node_->get_logger(), "Using default kp value of [%s]", std::to_string(kp).c_str());
+    }
+
+    if (_sdf->HasElement("i")){
```

Comment:
I would call this `ki`

Reply:
https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6